.github/workflows/ci.yml

```diff
@@ -40,13 +40,17 @@ jobs:
   docs:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
```

Comment:
Could you also update the action/setup-python on line 27 to a newer version?

Reply:
done in 1d5f23c